app/main.py

```diff
@@ -1,2 +1,76 @@
+class Node:
+    def __init__(self, key: int or float, value: int or float) -> None:
```

Comment:
Rewrite your type hinting to use `|` instead of `or` according to [PEP 604](https://peps.python.org/pep-0604/).